.github/workflows/sdk-api-doc.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           # See the `pathspec`-documentation for git
           # - https://git-scm.com/docs/git-add#Documentation/git-add.txt-ltpathspecgt82308203
           # - https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec
-          file_pattern: docs/sdk/*.md,packages/sdk/src
+          file_pattern: docs/sdk/*.md packages/sdk/src/packageMetadata.ts
```

Comment:
why we need to commit packageMetadata.ts?

Reply:
The packageMetadata.ts will be updated to the same version in package.json during `npm run setup`. This is to automate the version synchronization between them and prevent unit test break in CI.